記事更新処理(途中)

ManamiNomura opened the pull request.

ManamiNomura committed:
記事更新処理(途中)

EguchiAkira0908 commented on Model/ArticleModel.php:
```diff
@@ -137,6 +137,45 @@ public function index()
+    try {
+      // DBに接続
+      $pdo = $this->db_connect();
+      //SQL文 全データを投稿日時の降順で取得
```
コードはあってるけど、「全データ」ではないね。

EguchiAkira0908 commented on Model/ArticleModel.php:
```diff
@@ -166,11 +205,100 @@ public function search_index()
+      $articles = $pdo->prepare(
+        "SELECT * FROM articles WHERE id=$id"
+      );
+      // $articles->bindParam(':id', $id, PDO::PARAM_INT);
```
これはなんか意味があってとっておいてる？

ManamiNomura replied:
消しました

ManamiNomura committed:
修正

128行・147行・166行　全データを投稿日時の降順で取得→投稿日時の降順で取得

216行　      // $articles->bindParam(':id', $id, PDO::PARAM_INT);　　削除

EguchiAkira0908 commented:
おけです
